src/fluentd/deploy/fluentd-config.yaml.template

```diff
@@ -100,14 +100,6 @@ data:
       remove_keys log,time,stream,logSeverity,logPid,logSite
     </filter>
 
-    # Just for debugging
-    <filter fc-objectsnapshots>
-      @type stdout
-      <format>
-        @type json
-      </format>
-    </filter>
```

Comment:
Why remote it?

Reply:
Actually this log makes my debugging harder since `fc-objectsnapshots` is very huge, which makes other userful logs difficult to see. If it is necessary, I can revert this change.

Reply:
If so let's just comment it out, 

---
In reply to: [391375514](https://github.com/microsoft/pai/pull/4164#discussion_r391375514) [](ancestors = 391375514)

Reply:
> If so let's just comment it out,
> 
> In reply to: [391375514](https://github.com/microsoft/pai/pull/4164#discussion_r391375514) [](ancestors = 391375514)

OK, done.